config/manager/kustomization.yaml

```diff
@@ -1,16 +1,8 @@
 resources:
 - manager.yaml
-
-generatorOptions:
-  disableNameSuffixHash: true
-
-configMapGenerator:
-- files:
-  - controller_manager_config.yaml
-  name: manager-config
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: gcr.io/k8s-staging-nfd/node-feature-discovery-operator
-  newTag: master
+  newName: ghcr.io/arangogutierrez/node-feature-discovery-operator
+  newTag: v0.2.0-226-g4008131c-dirty
```

Comment:
??

Reply:
Kustomize.... grrr my bad, forgot to restore here